### PR TITLE
chore: Update README.md

### LINKED ISSUE #18: Add screenshots to the README file
Add screenshots to the README file

**Would you like to work on the issue?**

No

### PR DESCRIPTION
**Pull request by aman-g0yaa**
Fixes #18 

Changes: 

Added screenshots from the app .

- [x] Run unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into a single commit by squashing them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,22 @@ First time contributors can read [CONTRIBUTION.md](https://github.com/immadisair
 
 7. If the app was built successfully, you can test it by running it on either a real device or an emulated one by going to _Run>Run 'app'_ or pressing the Run icon in the toolbar.
 
+### Screenshots 
+<table>
+  <tr>
+    <td><img src="https://user-images.githubusercontent.com/51189244/60667450-321c4d80-9e87-11e9-84c9-d14485a39bda.jpeg" height = "480" width="270"></td>
+    <td><img src="https://user-images.githubusercontent.com/51189244/60667464-37799800-9e87-11e9-986e-c3c2a936b42a.jpeg" height = "480" width="270"></td>
+    <td><img src="https://user-images.githubusercontent.com/51189244/60667460-35173e00-9e87-11e9-9127-5a0e6fac9343.jpeg"" height = "480" width="270"></td>
+  </tr>
```
Comment (immadisairaj) [inline README.md]: I guess there is no need of having the same kind of pictures again. Can you please remove this.

**Review by immadisairaj — CHANGES_REQUESTED** [review]

**Comment by immadisairaj** [comment]
@amangoyal2339 try to squash commits into one as this is not a big issue

**Comment by immadisairaj** [comment]
> @amangoyal2339 try to squash commits into one as this is not a big issue

can you please do this

**Review by immadisairaj — APPROVED** [review]

**Review by immadisairaj — CHANGES_REQUESTED** [review]
> @amangoyal2339 try to squash commits into one as this is not a big issue

can you please do this

**Comment by immadisairaj** [comment]
> * [x]  If you have multiple commits please combine them into a single commit by squashing them.

@amangoyal2339 you have checked this but still you have multiple commits

**Comment by aman-g0yaa** [comment]
@immadisairaj I will squash them asap.

**Pull request closed without merging**